platform/android/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ MapLibre welcomes participation and contributions from everyone. Please read [`C
 ## main
 
 ### ✨ Features and improvements
+* Add support for the [`slice` expression](https://maplibre.org/maplibre-style-spec/expressions/#slice) ([#1113](https://github.com/maplibre/maplibre-native/pull/1133))
```

Comment:
@SiarheiFedartsou any reason why this is Android only?

Reply:
Yes, it should be quite easy to do that for iOS as well, but tbh I had no enough time and enough experience with iOS API of Maplibre in order to do that for iOS as well. I think it can be done in separate PR in the future. But if you can guide me to make this change available for iOS as well I would be happy to help.